app/views/merchant/dashboard/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Merchant Dashboard</h1>
+<h2>Orders</h2>
```

Comment:
these are just placeholders for the page. will be filled in later in other user stories